extensions/wikia/Search/classes/Services/CombinedSearchService.php

```diff
@@ -398,17 +484,14 @@ protected function getImage( $wikiId, $articleId ) {
 			if ( $row ) {
 				$dbName = $row->city_dbname;
 				if ( !empty( $dbName ) ) {
-					$db = wfGetDB( DB_SLAVE, [ ], $dbName ); // throws if database does not exits.
+					$db = wfGetDB( DB_SLAVE, [], $dbName ); // throws if database does not exits.
 					$imageServing = new \ImageServing(
-						[ $articleId ],
-						self::IMAGE_SIZE,
-						self::IMAGE_SIZE,
-						$db
+						[ $articleId ], self::IMAGE_SIZE, self::IMAGE_SIZE, $db
 					);
 					$isResult = $imageServing->getImages( 1 );
-					$images = isset( $isResult[ $articleId ] ) ? $isResult[ $articleId ] : false;
+					$images = isset( $isResult[$articleId] ) ? $isResult[$articleId] : false;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove the literal "false" boolean value. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1125)